examples/calendar/test/fmt.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
This is an IEF, do we need this file?

Reply:
If someone else wants to test out the circuit independently without running the entire file, this file helps a lot. I strongly recommend to keep it.

Reply:
Tests out the fixed merkle tree.